core/modules/utils/dom/dom.js

```diff
@@ -213,7 +213,7 @@ exports.addEventListeners = function(domNode,events) {
 				handler = eventInfo.handlerObject;
 			}
 		}
-		domNode.addEventListener(eventInfo.name,handler,false);
+		domNode.addEventListener(eventInfo.name,handler,!!eventInfo.capture);
```

Comment:
It's been a bit so I need to get my memory to work again :).

~~Capture is required because we need to have the option to intercept the event before it is consumed by something else (a KeyboardWidget or the iFrame of the framed editor). And it's set...~~

Scratch that, it used to work by using capture but at some point I rewrote it to not use capture. I'll fix this tomorrow and try the test cases I've enumerated to make sure it is still working.

Reply:
@saqimtiaz Pushed a commit that removes this obsolete change and also checked all my test cases - they still work!

Reply:
@EvidentlyCube looks good, and thank you for your patience.